app/models/html_attachment.rb

```diff
@@ -104,8 +104,23 @@ def publishing_api_presenter
     PublishingApi::HtmlAttachmentPresenter
   end
 
+  def invalid_contact_messages
+    errors.details[:body]
+      .select { |detail| detail[:error] == :invalid_contact }
+      .map { |detail| "Contact #{detail[:contact_id]} doesn't exist" }
+  end
+
 private
 
+  def skip_contact_validation?
+    return true if instance_variable_get(:@created_during_draft)
```

Comment:
Isn't this made redundant by line 118 below? 🤔 I'd have thought at the point the HTML attachment is created in `process_associations_after_save` it should be responding to `.draft?` with `true`?